ee/tables/nix_env/upgradeable/upgradeable.go

```diff
@@ -0,0 +1,144 @@
+//go:build linux
+// +build linux
+
+package nix_env_upgradeable
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/ee/allowedcmd"
+	"github.com/kolide/launcher/ee/dataflatten"
+	"github.com/kolide/launcher/ee/tables/dataflattentable"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+const allowedCharacters = "0123456789-"
```

Comment:
Is this is for uid, is `-` valid?

Reply:
I should take it out. I did have it for some _compatibility_, but it's very unlikely there will be a real user with a negative user id.